File: leetcode/solved/0771.jewels_and_stones.py
from collections import Counter

class Solution:
    def numJewelsInStones(self, jewels: str, stones: str) -> int:
        jewels_and_stones = Counter(stones)
        return sum(jewels_and_stones[k] for k in jewels)

if __name__ == '__main__':
    test = Solution()
    result = test.numJewelsInStones("aA", "aAAbbbb")
    print(result)

File: leetcode/0771.jewels_and_stones.cc
class Solution {
public:
    int numJewelsInStones(string jewels, string stones) {
        int jewels_count = 0;
        for(int i=0; i<stones.length(); i++) {
            if (jewels.find(stones[i]) != string::npos) {
                jewels_count += 1;
            }
        }
        return jewels_count;
    }
};